luminoso_api/client.py

```diff
@@ -125,7 +125,7 @@ def _get_token_auth(username, password, token, token_file, root_url):
             if username is None:
                 username = os.environ['USER']
             if password is None:
-                password = getpass('Password for %s: ' % username)
+                password = getpass(('Password for %s: ' % username).encoding('utf8'))
```

Comment:
Ah, I can see that the linked code snippet works around this issue by testing for both the OS and the Python version.  You probably need to do the same (although note that we already have the `compat` module here, into which that code should probably go).